xml/kernel_modules.xml

```diff
@@ -0,0 +1,179 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE chapter
+[
+  <!ENTITY % entities SYSTEM "entity-decl.ent">
+    %entities;
+]>
+<!-- Converted by suse-upgrade version 1.1 -->
+<chapter xmlns="http://docbook.org/ns/docbook" xmlns:xi="http://www.w3.org/2001/XInclude" xmlns:xlink="http://www.w3.org/1999/xlink" version="5.0" xml:id="cha.mod">
+ <title>Managing Kernel Modules</title>
+ <info>
+  <dm:docmanager xmlns:dm="urn:x-suse:ns:docmanager">
+   <dm:bugtracker></dm:bugtracker>
+   <dm:translation>yes</dm:translation>
+  </dm:docmanager>
+ </info>
+ <para>
+   Although Linux is a monolithic kernel, it can be extended using kernel
+   modules. These are special objects that can be inserted into the kernel and
+   removed on demand. In practical terms, kernel modules make it possible to
+   add and remove drivers and interfaces that are not included in the kernel
+   itself. Linux provides several commands for managing kernel modules.
+ </para>
+ <sect1 xml:id="sec.mod.lsmod">
+  <title>Listing Loaded Modules with lsmod and modinfo</title>
+  <para>
+    Use the <command>lsmod</command> command to view what kernel modules are
+    currently loaded. The output of the command may look as follows:
+  </para>
+  <screen>$ lsmod
```

Comment:
Please always use &prompt.user; (or, if applicable &prompt.root;) rather than some "custom" prompt. This also applies to other screens in this document.